### ISSUE TITLE
possible get a list of matching categories and ids given a string?

### ISSUE THREAD

**Issue by chovy** [benbuckman/nodejs-ebay-api#51]
Is this possible with this API?

**Comment by ajay2507** [comment]
@chovy  You can look into this node module https://github.com/ajay2507/ebay-node-api

**Comment by bhushankumarl** [comment]
@chovy 
Alternative is here https://www.npmjs.com/package/ebay-node-client#get-category-suggestions

**Comment by kroleg** [comment]
@chovy Nope. You linked new(JSON) API. This package is for old (XML) API
